src/Repository.js

```diff
@@ -118,8 +120,22 @@ export default class Repository {
   }
 
   buildPackageGraph() {
-    this._packages = PackageUtilities.getPackages(this);
-    this._packageGraph = PackageUtilities.getPackageGraph(this._packages);
+    const packages = PackageUtilities.getPackages(this);
+    const packageGraph = PackageUtilities.getPackageGraph(packages, false, this.lernaJson.useGitVersion
+      ? new GitVersionParser(this.lernaJson.gitVersionPrefix)
```

Comment:
I'd prefer the version parser be instantiated outside of the `getPackageGraph()` arguments, this ternary initially made me think you had forgotten the closing paren. Also, the version parser could be reused in the subsequent version serializer loop.

```js
const { useGitVersion, gitVersionPrefix } = this.lernaJson;
// FIXME: should be destructured from command.options to support CLI flag overrides

const versionParser = useGitVersion && new GitVersionParser(gitVersionPrefix);
const packages = PackageUtilities.getPackages(this);
const packageGraph = PackageUtilities.getPackageGraph(packages, false, versionParser);

if (useGitVersion) {
  packages.forEach((pkg) => {
    pkg.versionSerializer = new VersionSerializer({
      monorepoDependencies: packageGraph.get(pkg.name).dependencies,
      versionParser,
    });
  });
}
```